Core/Config.cpp

```diff
@@ -443,9 +435,7 @@ static ConfigSetting graphicsSettings[] = {
 #endif
 
 	// TODO: Replace these settings with a list of options
-	ConfigSetting("PartialStretch", &g_Config.bPartialStretch, &DefaultPartialStretch, true, true),
```

Comment:
That's right but only for the keyboard Blackberry 10 devices (there's only 5 of them): Q5, Q10, DAC, Classic, Passport

Inside a 1:1 aspect ratio looks bad without some zoom.